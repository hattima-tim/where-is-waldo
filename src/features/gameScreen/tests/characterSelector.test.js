import { render } from "@testing-library/react";
import CharacterSelector from "../characterSelector";

jest.mock("../../onBoardScreen/characters", () => () => {
  return (
    <>
      <p>Cacodemon</p>
      <p>Bill Cipher</p>
      <p>Courage</p>
    </>
  );
});

test("CharacterSelector component renders correctly", () => {
  const { container } = render(<CharacterSelector />);

  expect(container).toMatchSnapshot();
});
