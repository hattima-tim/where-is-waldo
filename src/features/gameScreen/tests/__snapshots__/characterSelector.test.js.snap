// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CharacterSelector component renders correctly 1`] = `
<div>
  <div
    class="flex"
  >
    <img
      alt="targetCircle"
      class="max-h-16 w-16"
      src="https://res.cloudinary.com/du3oueesv/image/upload/v1673284323/Where%27s%20Waldo/Where_s_Waldo_dot_k9q85x-media_lib_thumb_i9xieg.png"
    />
    <div
      class="z-10 w-72 rounded-md bg-[#17134d] p-4 text-white transition-all"
    >
      <p>
        Cacodemon
      </p>
      <p>
        Bill Cipher
      </p>
      <p>
        Courage
      </p>
    </div>
  </div>
</div>
`;
